react/features/base/conference/functions.js

```diff
@@ -200,5 +204,10 @@ export function sendLocalParticipant(
     email && conference.sendCommand(EMAIL_COMMAND, {
         value: email
     });
+
+    if (features && features['screen-sharing'] === 'true') {
+        conference.setLocalParticipantProperty('features_screen-sharing', true);
```

Comment:
Haha, good question :) I will rename it

Reply:
When looking at it again, so from token we get screen-sharing and features_screen-sharing is like features/screen-sharing. As this goes in xmpp I didn't want to add some special chars, maybe '/' is ok, but just in case.